Tools/PrivateRepoLib.ps1

```diff
@@ -0,0 +1,229 @@
+
+class ARMObject
```

Comment:
document what this file does at the top of the file.

Reply:
Added comment at the top of the script documenting what the purpose of this script is.